nodes/nomos-node/config.yaml

```diff
@@ -49,3 +53,12 @@ http:
   backend:
     address: 0.0.0.0:8080
     cors_origins: []
+
+da:
+  da_protocol:
+    num_attestations: 1
+  backend:
+    max_capacity: 10
+    evicting_period: 
+      secs: 3600
+      nanos: 0
```

Comment:
Same, this is not from this pr or is it?

Reply:
This is from this PR, had to add these changes for config to work.
Added these changes to PR because they are small, these are separate commits:
![image](https://github.com/logos-co/nomos-node/assets/2364987/de19cea2-25f6-4e50-bd67-c4fd3b7d0d1e)